### PR TITLE
feat: add copyUserURLs plugin

### PR DESCRIPTION
**Pull request by castdrian**
this adds a button to copy user urls, if this is useless just close, I mainly made it for myself since I use user urls a lot

### CODE REVIEW

**Comment by CREAsTIVE** [comment]
why not a plugin?

File: src/plugins/copyUserURLs.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+interface UserContextProps {
+    channel: Channel;
+    guildId?: string;
+    user: User;
+}
+
+const UserContextMenuPatch: NavContextMenuPatchCallback = (children, { user }: UserContextProps) => () => {
+    function copyUserURL() {
+        Clipboard.copy(`<https://discord.com/users/${user.id}>`);
+        Toasts.show({
```
Comment (Vendicated) [inline src/plugins/copyUserURLs.tsx:35]: just use copyWithToast instead

Comment (castdrian) [reply src/plugins/copyUserURLs.tsx]: just removed the toast and cleaned it up, copy user ID doesn't have any confirmation either, dunno what my 2 AM brain was thinking

File: src/plugins/copyUserURLs.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+interface UserContextProps {
+    channel: Channel;
+    guildId?: string;
+    user: User;
+}
+
+const UserContextMenuPatch: NavContextMenuPatchCallback = (children, { user }: UserContextProps) => () => {
+    children.push(
+        <Menu.MenuItem
+            id="copy-user-url"
```
Comment (Vendicated) [inline src/plugins/copyUserURLs.tsx:35]:
```suggestion
            id="vc-copy-user-url"
```

**Pull request merged**